bitnami/keycloak/Chart.yaml

```diff
@@ -26,4 +26,4 @@ name: keycloak
 sources:
   - https://github.com/bitnami/bitnami-docker-keycloak
   - https://github.com/keycloak/keycloak
-version: 9.4.1
+version: 9.4.2
```

Comment:
Please bump a minor version `9.5.0`

Reply:
Fixed both the readme and the missing .Release.Namespace, thank you for the review!